src/Development/IDE/Plugin.hs

```diff
@@ -0,0 +1,33 @@
+
+module Development.IDE.Plugin(Plugin(..), codeActionPlugin) where
+
+import Data.Default
+import Development.Shake
+import Development.IDE.LSP.Server
+
+import           Language.Haskell.LSP.Types
+import Development.IDE.Core.Rules
+import qualified Language.Haskell.LSP.Core as LSP
+import Language.Haskell.LSP.Messages
+
+
+data Plugin = Plugin
+    {pluginRules :: Rules ()
+    ,pluginHandler :: PartialHandlers
+    }
+
+instance Default Plugin where
```

Comment:
Not a big fan of `Data.Default`. Can we just export `defPlugin` instead?

Reply:
We're already using Data.Default for the rest of this API. I wanted to stay consistent.

Reply:
Fair enough